src/foam/dao/DAOCopySink.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ foam.CLASS({
+  package: 'foam.dao',
+  name: 'DAOCopySink',
+  extends: 'foam.dao.AbstractSink',
+
+  documentation: 'Puts all objects in the sink into a different DAO',
+
+  properties: [
+    {
+      class: 'Class',
+      name: 'of'
+    },
+    {
+      class: 'Object',
+      name: 'outputDAO',
+      javaType: 'foam.dao.DAO',
+    }
+  ],
+
+  methods: [
+    {
+      name: 'put',
+      code: function put(o, sub) {
+        this.outputDAO.put(o);
+      },
+      javaCode: 'getOutputDAO().put((foam.core.FObject) obj);'
```

Comment:
@kgrgreer think we should clone.

Reply:
Or @nick-prat perhaps a flag, if there is a chance both daos will update.  I'm assuming at the moment, one is copy for read/review.

Reply:
@jlhughes I think a flag for cloning is a good idea.